### PR TITLE
Consolidate and refactor logging configuration code

### PR DESCRIPTION
**Pull request by shanemcd**
##### SUMMARY
Once this merges, we can also delete the _new_ code that was added in https://github.com/ansible/awx-operator/pull/1026

##### ISSUE TYPE

 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME

 - API

### CODE REVIEW

File: awx/settings/defaults.py [hunk 1]
```diff
@@ -821,86 +821,25 @@ def IS_TESTING(argv=None):
...
-            'formatter': 'dispatcher',
-        },
-        'wsbroadcast': {
-            # don't define a level here, it's set by settings.LOG_AGGREGATOR_LEVEL
-            'class': 'logging.handlers.WatchedFileHandler',
-            'filters': ['require_debug_false', 'dynamic_level_filter', 'guid'],
-            'filename': os.path.join(LOG_ROOT, 'wsbroadcast.log'),
-            'formatter': 'simple',
-        },
-        'celery.beat': {'class': 'logging.StreamHandler', 'level': 'ERROR'},  # don't log every celerybeat wakeup
```
Comment (shanemcd) [inline awx/settings/defaults.py:868]: This was unused so I just deleted it.

File: awx/settings/defaults.py [hunk 2]
```diff
@@ -821,86 +821,25 @@ def IS_TESTING(argv=None):
...
-            'formatter': 'simple',
-        },
-        'management_playbooks': {
-            'level': 'DEBUG',
-            'class': 'logging.handlers.WatchedFileHandler',
-            'filters': ['require_debug_false'],
-            'filename': os.path.join(LOG_ROOT, 'management_playbooks.log'),
-            'formatter': 'simple',
-        },
-        'system_tracking_migrations': {
```
Comment (shanemcd) [inline awx/settings/defaults.py:884]: This also seemed to be unused so I deleted it.

File: awx/settings/defaults.py [hunk 3]
```diff
@@ -821,86 +821,25 @@ def IS_TESTING(argv=None):
...
-        },
-        'rbac_migrations': {
-            'level': 'WARNING',
-            'class': 'logging.handlers.WatchedFileHandler',
-            'filters': ['require_debug_false'],
-            'filename': os.path.join(LOG_ROOT, 'tower_rbac_migrations.log'),
-            'formatter': 'simple',
-        },
-        'job_lifecycle': {
-            'level': 'DEBUG',
```
Comment (shanemcd) [inline awx/settings/defaults.py:899]: We were explicitly setting debug here, but almost all of our other handlers have a comment that says "don't define a level here, it's set by settings.LOG_AGGREGATOR_LEVEL". @fosterseth Do you know if this was intentional? If so, do you remember the reason?

Comment (fosterseth) [reply awx/settings/defaults.py:899]: can't remember, but I do know the job lifecycle logging uses debug level, so maybe setting `level` here was to override log_aggregator_level.

We can (and probably should) just change this to `info`
```python
        logger_job_lifecycle.debug(msg, extra=extra)
```
https://github.com/fosterseth/awx/blob/devel/awx/main/models/unified_jobs.py#L1552

**Review by AlanCoding — APPROVED** [review]

**Pull request merged**